src/sidebar/components/test/user-menu-test.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+const { createElement } = require('preact');
+const { shallow } = require('enzyme');
+
+const UserMenu = require('../user-menu');
+const MenuItem = require('../menu-item');
+
+describe('UserMenu', () => {
+  let fakeAuth,
+    fakeBridge,
+    fakeIsThirdPartyUser,
+    fakeOnLogout,
+    fakeProfileBridgeEvent,
+    fakeServiceConfig,
+    fakeServiceUrl,
+    fakeSettings;
+
+  const createUserMenu = () => {
+    return shallow(
+      <UserMenu
+        auth={fakeAuth}
+        bridge={fakeBridge}
+        onLogout={fakeOnLogout}
+        serviceUrl={fakeServiceUrl}
+        settings={fakeSettings}
+      />
+    ).dive(); // Dive needed because this component uses `withServices`
+  };
+
+  const findMenuItem = (wrapper, labelText) => {
+    return wrapper
+      .find(MenuItem)
+      .filterWhere(n => n.prop('label') === labelText);
+  };
+
+  beforeEach(() => {
+    fakeAuth = {
+      displayName: 'Eleanor Fishtail',
+      status: 'logged-in',
+      userid: 'acct:eleanorFishtail@hypothes.is',
+      username: 'eleanorFishy',
+    };
+    fakeBridge = { call: sinon.stub() };
+    fakeIsThirdPartyUser = sinon.stub();
+    fakeOnLogout = sinon.stub();
+    fakeProfileBridgeEvent = 'profile-requested';
+    fakeServiceConfig = sinon.stub();
+    fakeServiceUrl = sinon.stub();
+    fakeSettings = {
+      authDomain: 'hypothes.is',
+    };
+
+    UserMenu.$imports.$mock({
+      '../util/account-id': {
+        isThirdPartyUser: fakeIsThirdPartyUser,
+      },
+      '../service-config': fakeServiceConfig,
+      '../../shared/bridge-events': {
+        PROFILE_REQUESTED: fakeProfileBridgeEvent,
+      },
+    });
+  });
+
+  afterEach(() => {
+    UserMenu.$imports.$restore();
+  });
+
+  describe('profile menu item', () => {
+    context('first-party user', () => {
+      beforeEach(() => {
+        fakeIsThirdPartyUser.returns(false);
+        fakeServiceUrl.returns('profile-link');
+      });
+
+      it('should be enabled', () => {
+        const wrapper = createUserMenu();
+
+        const profileMenuItem = findMenuItem(wrapper, fakeAuth.displayName);
+        assert.isUndefined(profileMenuItem.prop('isDisabled'));
```

Comment:
`assert.notOk` (ie. "falsey") might be more appropriate here, since passing `isDisabled={false}` explicitly would also be correct.